src/main/java/com/google/ehub/data/CommentData.java

```diff
@@ -0,0 +1,14 @@
+package com.google.ehub.data;
+
+public class CommentData {
```

Comment:
Each class and public method should have a javadoc that describes the purpose of the class.